demo_nodes_cpp/CMakeLists.txt

```
@@ -71,6 +71,8 @@ add_library(topics_library SHARED
   src/topics/talker.cpp
   src/topics/talker_loaned_message.cpp
   src/topics/talker_serialized_message.cpp
+  src/topics/temperature_publisher.cpp
+  src/topics/temperature_subscriber.cpp
```

```suggestion
  src/topics/content_filtering_publisher.cpp
  src/topics/content_filtering_subscriber.cpp
```

sorry i was not clear on this, i think files, classes and components can be general names for content filtering, so that user can tell these are related to content filtering.